client/state/selectors/get-pending-revisions-count.js

```diff
@@ -0,0 +1,18 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+
+const getPendingRevisionsCount = createSelector(
```

Comment:
After the recent `cachedSelector` saga it might be worth revisiting this - does it benefit from the caching? I'm not sure - I think I used `createSelector` naively before knowing the deeper details of it's logic.